server/src/main/java/org/elasticsearch/index/fielddata/MultiGeoValues.java

```diff
@@ -144,5 +155,56 @@ public static GeoShapeValue missing(String missing) {
     public interface GeoValue {
         double lat();
         double lon();
+        BoundingBox boundingBox();
+    }
+
+    public static class BoundingBox {
```

Comment:
More curiosity than anything, why do we need all these values instead of just upper-left and lower-right?

Reply:
because of the the bounding-box is resolved when it is sensitive to longitudinal wrapping https://github.com/elastic/elasticsearch/blob/master/server/src/main/java/org/elasticsearch/search/aggregations/metrics/InternalGeoBounds.java#L214

Reply:
ahh, gotcha.  Good ol wrapping :)